example/about.md

```diff
@@ -7,9 +7,9 @@ permalink: /about/
 This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
 
 You can find the source code for the Jekyll new theme at:
-{% include icon-github.html username="jekyll" %} /
+<span class="svg-icon github"></span> /
 [minima](https://github.com/jekyll/minima)
```

Comment:
You missed the link to `[jekyll-organisation]`. Now its just `[icon] / [minima-repo]`.
Same mistake below